oleaut32.go

```diff
@@ -187,6 +198,182 @@ func VariantBSTRToString(value *VAR_BSTR) string {
 	return BSTRToString(value.bstrVal)
 }
 
+func (v *VARIANT) LVal() int32 {
```

Comment:
This should be called `MustLong` and use `panic` instead of `log.Fatal` to allow recovery.